src/sentry/remote_config/docs/api.md

```diff
@@ -44,16 +68,30 @@ Retrieve the DSN's configuration.
 
 ### Set Configuration [POST]
 
-Set the DSN's configuration.
+Set the DSN's configuration. A hash value of `null` is provided if the object does not exist. If the API finds an existing configuration object an error is returned. If the hash value is provided then it will be compared against the hash of the remote value. A new hash is always returned on successful write. If the hashes do not match an error is returned.
```

Comment:
Just to clarify, is the hash here used to handle concurrent writes? e.g.

* users A and B have configuration `1`
* user B updates the configuration first, remote now has config w/ hash `2`
* user A attempts to update but fails because their client hash is `1` !== server hash `2`

Reply:
Not concurrent writes just stale writes.  I'm not sure if I want to include it still.  I'm trying to avoid a situation where a stale session overwrites lots of changes.  I'm not sure its worth it atm.

Reply:
i think it's fair to drop it for now. We only need to build what's minimum to get things working end to end to start and this is extra on that front.

Also, if we look at the previous api design which could be our end-state design... that is very different, so there's time to decide how we would solve for stale and/or concurrent writes at that point. We don't need to have a solution for it with this version of the api

Reply:
I'd even say... we shouldn't need to write down the `features` field yet if it'll be unimplemented by the UI, BE, and/or SDK.

Reply:
It should be implemented in the UI --mulitple key, value pairs.  The point of the previous API design was to allow the Web UI to move fast.  It was never the final design.  I think the new API is simple enough that should be quick for you to iterate on while also providing the full feature set of the final product.

Reply:
Billy will be implementing the `features` field on the SDK this week and I've already pushed a branch handling it on the API side.

Reply:
ah, ok
i'll put it into the ui asap too